harvester_ng/csw/functions.py

```diff
@@ -72,16 +75,6 @@ def validate_datasets(row):
     row['validation_errors'] = []
 
 
-# we need a way to save as file using an unique identifier
```

Comment:
this is defined already in helpers, I changed the uses of the one from this file to the one in the helpers file